src/features/Applet/Popups/AppletPasswordPopup/AppletPasswordPopup.types.ts

```diff
@@ -0,0 +1,16 @@
+import { Encryption } from 'redux/modules';
+import { AppletPasswordForm } from 'features/Applet/Password/EnterAppletPassword/EnterAppletPassword.types';
+
+export enum AppletPasswordPopupType {
+  CREATE = 'create',
```

Comment:
I think it's better to use Create, Enter (capitalize only first letter)
Like here: https://www.typescriptlang.org/docs/handbook/enums.html